sdk/appconfiguration/spring-cloud-azure-appconfiguration-config-web/src/main/java/com/azure/spring/cloud/config/web/implementation/config/AppConfigurationWebAutoConfiguration.java

```diff
@@ -29,7 +29,7 @@
 @EnableConfigurationProperties(AppConfigurationProperties.class)
 @RemoteApplicationEventScan
 @ConditionalOnBean(AppConfigurationRefresh.class)
-public class AppConfigurationWebAutoConfiguration {
+class AppConfigurationWebAutoConfiguration {
```

Comment:
Will this AutoConfiguration class referenced in the `spring.factories`. For a Spring AutoConfiguration class, which users can exclude in their application, we should not hide them in the `impl` package. We can move all the bean definition in a Configuration class, and Import that Configuation in this `AutoConfiguration`. Hence the AutoConfiguration won't have any bean definitions.

Reply:
Moved outside of impl.